### ISSUE TITLE
🐛 BUG: Dev Server should return unmodified versions of "./public" files

### ISSUE THREAD

**Issue by johnhenry** [snowpackjs/astro#749]
### What package manager are you using?

npm

### What operating system are you using?

Mac

### Describe the Bug

Public files are modified in various ways including removing comments and overriding rile content.
I suspect that this has something to do with settings of the underlying dev server and may be related to #697.

Note: astro version 0.17.2

### Steps to Reproduce

1. Run `npm init astro` and select generic template
1. Add file "/public/index.mjs" with content "console.log('mjs'); // comment".
1. Add file "/public/index.js" with content "console.log('js'); // comment".
1. Run dev server with `npm start`
1. Error! Visit http://localhost:3000/index.mjs and observe "Not Found" error.
1. Error! visit http://localhost:3000/index.js and observe content for "index.mjs", but with comment removed.
1. Note: building the application with `npm run build` appears to work properly.

**Comment by drwpow** [comment]
Agreed! This is a current limitation / quirk of how our build pipeline works now. In fact, some of our examples even show putting `.scss` files in `/public` when they shouldn’t be there. This will be fixed in our new upcoming compiler / before 1.0! But it may not happen sooner than that.
